jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
Please, share your thoughts at #1795, too.

It seems that there will be no easy consensus on this topic, so let's drop the debugger rewrite for now (the patch already became bitrotten anyway). But what abut the `lit`-related part (the topmost commit)?